helm/up9/values.yaml

```diff
@@ -15,6 +15,7 @@ up9BaseRepositoryPath: gcr.io/up9-docker-hub
 imagesBranch: master    # all components will use this branch
 imagesTag: ":latest"      # all components will use this tag
 standalone: false
+priorityClassName:
```

Comment:
can you please add a comment here --> # priority class for all UP9 pods (optional, use default if not specified here)